DevAcademy/Scenes/PlaceDetail/PlaceDetailViewState.swift

```diff
@@ -16,18 +16,54 @@ struct PlaceDetailViewState: DynamicProperty {
     }
 
     var placeTitle: String {
-        place.properties.nazev
+        place.properties.name
     }
 
     var placeType: String {
-        place.properties.druh.rawValue
+        place.properties.kind.rawValue
     }
 
-    var placeImageUrl: URL {
-        place.properties.obrId1
+    var placeAddress: String {
+        guard let street = place.properties.street, let identificationNumber = place.properties.identificationNumber else {
+            return ""
+        }
+
+        return "\(street) \(identificationNumber)"
+    }
+
+    var placeEmail: String? {
```

Comment:
mozna by bylo presnejsi pojmenovani `placeEmailScheme`, at te to nekdy v budoucnu nezmate, ze tam budes ocekavat ciste ten email toho mista. Stejne tak u toho `placePhone`.😉